src/frontend/src/env/networks.icrc.env.ts

```diff
@@ -385,6 +387,35 @@ const CKXAUT_IC_DATA: IcCkInterface | undefined = nonNullish(CKERC20_PRODUCTION_
 		}
 	: undefined;
 
+/**
+ * Additional ICRC tokens from JSON file
+ */
+const mapIcrcData = (
```

Comment:
may you please create some tests for this one?

and shouldn't this go somewhere in utils? or is it done in the same way elsewere?

Reply:
done